x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldMapper.java

```diff
@@ -56,81 +73,118 @@ protected String contentType() {
         return CONTENT_TYPE;
     }
 
-    public static class Builder extends FieldMapper.Builder<Builder> {
-
+    public static class Builder extends ParametrizedFieldMapper.Builder {
+
+        private final Parameter<Map<String, String>> meta = new Parameter<>(
```

Comment:
You should be able to use `Parameter.metaParam()` here

Reply:
good one, the feature branch was slightly behind, I will merge master in.